src/main/java/com/swervedrivespecialties/exampleswerve/RobotContainer.java

```diff
@@ -0,0 +1,70 @@
+package com.swervedrivespecialties.autonrefresh;
+
+import com.swervedrivespecialties.autonrefresh.subsystems.DrivetrainSubsystem;
+import com.swervedrivespecialties.autonrefresh.util.AutonChooser;
+import com.swervedrivespecialties.autonrefresh.util.BeakXboxController;
+import com.swervedrivespecialties.autonrefresh.util.util;
+import com.swervedrivespecialties.exampleswerve.commands.DriveCommand;
+
+public class RobotContainer {
+
+    public DrivetrainSubsystem drive = new DrivetrainSubsystem.getinstance(); 
```

Comment:
```suggestion
    public DrivetrainSubsystem drive = DrivetrainSubsystem.getinstance(); 
```
new is to call the constructor of a class. getinstance() isn't a constructor, it's a static function returning an instance (which comes from calling  a constructor, but it's a static instance that's a class-level variable.